program-runtime/src/loaded_programs.rs

```diff
@@ -347,16 +347,21 @@ impl LoadedProgram {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct ProgramRuntimeEnvironments {
+    /// Globally shared RBPF config and syscall registry
+    pub program_runtime_environment_v1: Arc<BuiltinProgram<InvokeContext<'static>>>,
```

Comment:
we can take out the "environment" in the name here and then have e.g.
"environments.program_runtime_v1"